addons/police/functions/fnc_policeStationAlarm.sqf

```diff
@@ -0,0 +1,28 @@
+#include "script_component.hpp"
+/*
+ * Author: 3Mydlo3
+ * Function handles police station alarm messages.
+ *
+ * Arguments:
+ * 0: Object which triggered alarm <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player] call afsk_police_fnc_policeStationAlarm
+ *
+ * Public: No
+ */
+
+params ["_object"];
+
+titleText [LSTRING(Bad_Idea), "PLAIN", 5];
+
+private _nearestTown = [_object] call MDL_SK_fnc_nearest_town;
```

Comment:
Leftover function from legacy codebase?

Reply:
Yes, I copy the code from there and I try to organize everything in the new way. Almost nothing from legacy code gets through to new version but such mistakes probably will happen for some time.